java_shared.gradle

```diff
@@ -2,6 +2,7 @@ sourceCompatibility = 1.7
 targetCompatibility = 1.7
 
 javadoc {
+    failOnError false
```

Comment:
is this related to the PR?

Reply:
Yes
I'm still trying to figure why the javadoc fails the build.

Reply:
removed.